src/client/Microsoft.Identity.Client/AuthenticationResultMetadata.cs

```diff
@@ -70,5 +70,10 @@ public AuthenticationResultMetadata(TokenSource tokenSource)
         /// Contains the Outcome of the region discovery if Region was used.
         /// </summary>
         public RegionDetails RegionDetails { get; set; }
+
+        /// <summary>
+        /// Contains broker telemetry in JSON format
```

Comment:
>broker [](http://example.com/codeflow?start=21&length=6)

Technical telemetry could also be coming from runtime. Should we make the comment more generic and just leave it at "Contains telemetry in JSON format"